sql/parser2/eval_test.go

```diff
@@ -134,7 +132,6 @@ func TestEvalExpr(t *testing.T) {
 		// Comparisons against NULL result in NULL, except for the null-safe equal.
```

Comment:
I have another set of changes to expression evaluation which fixes this and adds more tests, but when I included those changes (which are decently significant), github thought everything was new instead of just movement. So yes, you're right, it will be fixed shortly.